ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/qos.md

```diff
@@ -325,6 +340,11 @@ QOS Profile: **no_qos_trust**
 | ----------- | ------------ | ----- | ---------- | ------------------ |
 | 3 | 4 | disabled | - | - |
 
+**ECN Configuration**
```

Comment:
Please add some condition to avoid printing the empty table.

Reply:
Added logic to skip any ECN Configuration tables if there are no tx queues or if none of them have any ECN config.